sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/ReadFromKafkaDoFnTest.java

```diff
@@ -432,7 +432,7 @@ public void testProcessElementWhenTopicPartitionIsStopped() throws Exception {
             makeReadSourceDescriptor(consumer)
                 .toBuilder()
                 .setCheckStopReadingFn(
-                    new SerializableFunction<TopicPartition, Boolean>() {
+                    new CheckStopReadingFn() {
```

Comment:
sorry for being late. Let's revert the change in test to make sure the compatibility is tested